CHANGELOG.md

```diff
@@ -20,7 +20,7 @@
 -----------
 
 ### Internals
-* None.
+* All exceptions thrown out of Core are now of type 'Exception'. All use of std::runtime_error and std::logical_error etc. has stopped and the specialized error classes that beforehand were based on these are now based on Exception.
```

Comment:
That's not an "internal change"?

Reply:
There seem to be a bit confusion around what "internal" means. Is it internal to Core or internal to the Realm products?

Reply:
I read `Internal` as Internal to Core, i.e. changes that doesn't impact SDK implementations.